libdevcore/Log.cpp

```diff
@@ -43,7 +43,7 @@ void dev::simpleDebugOut(std::string const& _s, char const*)
 	static Mutex s_lock;
 	Guard l(s_lock);
 
-	cerr << _s << endl << flush;
+	cout << _s << endl << flush;
```

Comment:
You never employed that convention since this project started, and it made a lot more sense that way. I'll remove that so you can merge the fix to neth.